salt/utils/cloud.py

```diff
@@ -1388,21 +1388,19 @@ def deploy_windows(
                 salt.utils.smb.delete_directory("salttemp", "C$", conn=smb_conn)
         # Shell out to psexec to ensure salt-minion service started
         if use_winrm:
-            winrm_cmd(winrm_session, "sc", ["stop", "salt-minion"])
+            winrm_cmd(winrm_session, "net", ["stop", "salt-minion"])
```

Comment:
**question:** : Is `net` available by default on all windows platforms we support?

Reply:
yes